sharding/chainParametersHolder_test.go

```diff
@@ -6,8 +6,9 @@ import (
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	"github.com/ElrondNetwork/elrond-go-core/data/block"
 	"github.com/ElrondNetwork/elrond-go/config"
-	"github.com/ElrondNetwork/elrond-go/testscommon/epochNotifier"
+	"github.com/ElrondNetwork/elrond-go/sharding/mock"
```

Comment:
can you use one epoch start notifier from testscommon?
so we can gradually remove all other mocks

Reply:
done + removed other unused mocks and stubs